profiling/src/internal/observation/trimmed_observation.rs

```diff
@@ -183,4 +183,33 @@ mod tests {
             assert_eq!(*b, vec![1, 2]);
         }
     }
+
+    // cargo +nightly bolero test
+    // internal::observation::trimmed_observation::tests::fuzz_trimmed_observation -T 1min
```

Comment:
Not sure if we need this as a comment.  But a block comment explaining what you do and don't fuzz over might be nice